src/android/io/sqlc/SQLitePlugin.java

```diff
@@ -256,7 +264,15 @@ private SQLiteAndroidDatabase openDatabase(String dbname, String key, CallbackCo
             return mydb;
         } catch (Exception e) {
             // NOTE: NO Android locking/closing BUG workaround needed here
-            cbc.error("can't open database " + e);
+
+            if(selfHealingEnabled && e.getMessage().contains("file is encrypted or is not a database:")) {
+                logger.logError("Android ciphered database will be deleted to self heal: " + e.getMessage(), "SQLite", e);
```

Comment:
This should be a Warning to be consistent with the healing logs coming from other plugins, namely the Cache.